test/test_filters.py

```diff
@@ -136,3 +136,54 @@ def test_b(request, case):
     result = testdir.runpytest("-v", "-s")
     result.assert_outcomes(passed=2)
     result.stdout.re_match_lines([r"Hypothesis calls: 2$"])
+
+
+def test_operationId_filter_with_openapi(testdir):
```

Comment:
Could you, please, rename `operationId` to `operation_id` to follow naming convention in other tests?

Reply:
Yes, I modified it